proseco/tests/test_catalog.py

```diff
@@ -208,7 +208,7 @@ def test_call_args_attr():
                              'n_acq': 0,
                              'n_fid': 0,
                              'n_guide': 0,
-                             'obsid': 1,
+                             'obsid': 0,
```

Comment:
Using obsid=0 instead of obsid=1 ensures that the mica starcheck call will never accidentally happen.